docs/sources/tempo/troubleshooting/long-running-traces.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
great start, i would mention that:

- the `trace_idle_period` plays an important role here and can allow operators to adjust for longer running trace pipelines.
- we provide some data accuracy metrics under `tempo_warnings_total`. one indicates if a disconnected trace is flushed to the wal, the other indicates if a rootless trace is flushed to the wal.

https://github.com/grafana/tempo/blob/main/pkg/dataquality/warnings.go

this is the formula we use in cloud to generate %age of connected traces flushed to wal:

```
1 - sum(rate(tempo_warnings_total{reason="disconnected_trace_flushed_to_wal"}[5m])) / sum(rate(tempo_ingester_traces_created_total{}[5m]))
```

Reply:
This is handy to know, thanks for the pointers.